doc/flame_svg/svg.md

```diff
@@ -0,0 +1,13 @@
+<!-- markdownlint-disable MD013 -->
+<p align="center">
+  <a href="https://flame-engine.org">
+    <img alt="flame" width="200px" src="https://user-images.githubusercontent.com/6718144/101553774-3bc7b000-39ad-11eb-8a6a-de2daa31bd64.png">
+  </a>
+</p>
+<!-- markdownlint-enable MD013 -->
+
+# Flame SVG
```

Comment:
This should explain how to use the `SvgComponent`

Reply:
Where can I get that info?

Reply:
You can have a look at the code in the example code [here](https://examples.flame-engine.org/#/Svg_Svg%20Component), press `< >` in the upper right corner to get to the code.

Also this section: https://docs.flame-engine.org/1.4.0/flame/rendering/images.html#svg
should be moved to be under this package too.